python-modules/cis_profile/cis_profile/fake_profile.py

```diff
@@ -161,11 +161,16 @@ def login_method(self):
         p = ["Mozilla-LDAP", "google-oauth2", "github", "firefoxaccounts", "email"]
         return self.generator.random.choice(p)
 
-    def usernames(self, additional=True):
-        u = {"mozilliansorg": self.generator.user_name()}
-        if additional:
-            for _ in range(self.generator.random.randrange(0, 10)):
-                u[self.generator.words(nb=1)[0]] = self.generator.user_name()
+    def primary_username(self):
+        if self.generator.boolean():
+            return "r--{}".format(base64.urlsafe_b64decode(fake.uuid4()))
```

Comment:
is this meant to be `r--`?

Reply:
Yes. Maybe I should add a comment?
This is how the random usernames look like in DinoPark these days. The `r--` prefix is used to determine that this is a random username.

I can leave it out here. But it's something we need to figure out otherwise.

The use case is that our front-end wants to hide _random_ usernames in the main profile card. And I thought that's a rather sustainable solution.

Reply:
ah ok, yeah a comment wouldnt be bad